collector/test-data-uploader/src/main/kotlin/com/buildkite/test/collector/android/models/RunEnvironment.kt

```diff
@@ -46,7 +46,7 @@ internal data class RunEnvironment(
     companion object {
         // When bumping version, update VERSION_NAME to match new version
         // Used for uploading correct library version
-        const val VERSION_NAME = "0.2.0"
+        const val VERSION_NAME = "0.3.0"
```

Comment:
Just confidence checking that the versions in your readme don't need to match what's in the RunEnvironment / gradle files?

Reply:
Yes, that's correct. The version in README is latest released version. This upgrade is just setting up our next release version under which we'd upload our test/SNAPSHOT artifacts.